src/kokkos_abstraction.hpp

```diff
@@ -230,12 +248,10 @@ template <typename Function>
 inline void par_for(LoopPatternMDRange, const std::string &name, DevExecSpace exec_space,
                     const int &jl, const int &ju, const int &il, const int &iu,
                     const Function &function) {
-  Kokkos::parallel_for(
-      name,
-      Kokkos::Experimental::require(
-          Kokkos::MDRangePolicy<Kokkos::Rank<2>>(exec_space, {jl, il}, {ju + 1, iu + 1}),
-          Kokkos::Experimental::WorkItemProperty::HintLightWeight),
-      function);
+  Kokkos::parallel_for(name,
+                       par_policy(Kokkos::MDRangePolicy<Kokkos::Rank<2>>(
+                           exec_space, {jl, il}, {ju + 1, iu + 1}, {1, iu + 1 - il})),
+                       function);
```

Comment:
Not for now, but we should consider creating a par_for_internal, or par_for_leaf that iterates from NGHOST to nblock_dir+NGHOST.  Most of the physics routines likely use this range and this would make invocations in the main code more compact since loop extents wouldn't have to be passed.

Reply:
Closely related to #183 and #185